src/ui/features/dashboards/delete-dashboard.feature

```diff
@@ -0,0 +1,17 @@
+@regression
+Feature: Dashboards
+    
+    Background: 
```

Comment:
Background section pliable in case of common actions for several scenarios, you can combine it with other scenario on oder to have correct structure and Background usage

Reply:
same for src/ui/features/dashboards/edit-dashboard.feature

Reply:
Combined Edit and Delete tests in one feature using background. Thanks for the comment